faf-java-server-app/src/main/java/com/faforever/server/entity/Player.java

```diff
@@ -88,4 +99,16 @@ public void setGameState(PlayerGameState gameState) {
   public String toString() {
     return "Player(" + getId() + ", " + getLogin() + ")";
   }
+
+  @Value
+  public static class FraudReport {
+    int reporterId;
+    int gameId;
+    String name;
```

Comment:
Is name the reason for the report? Can we call it reasonForReport or something more descriptive?

Reply:
it's the player name, I'll adjust it to `playerName`.